blog/_posts/2018-04-19-scala-3.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: blog-detail
+post-type: blog
+by: Martin Odersky
+title: "Towards Scala 3"
+---
+
+Now that Scala 2.13 is only a few months away, it's time to consider the
+roadmap beyond it. It's been no secret that the work on [Dotty](https://github.com/lampepfl/dotty) over the
+last 5 years was intended to explore what a new Scala could look
+like. We are now at a stage where we can commit: Dotty will become
+Scala 3.0.
+
+Of course, this statement invites many follow-up questions. Here are
+some answers we can already give today. We expect there will be more
+questions and answers as things shape up.
+
+_When will it come out?_
+
+The intent is to publish the final Scala 3.0 soon after Scala 2.14. At the
+current release schedule (which might still change), that means early 2020.
+
+_What is Scala 2.14 for?_
+
+Scala 2.14's main focus will be on smoothing the migration to Scala
+3. It will do this by defining migration tools, shim libraries, and
```

Comment:
Markdown parsing in Jekyll is interpreting this as a bulleted list:

![Github screen shot](https://user-images.githubusercontent.com/358615/38991002-40728a90-43aa-11e8-8d15-214fae6db317.png)

Although viewing the diff in Github is highlighting the "`3.`" in red.

Reply:
nice catch!